### DISCUSSION TITLE
Announcing v8.2.1.0 for Firefox Beta

### DISCUSSION THREAD

**Discussion by paulrouget** [aclap-dev/video-downloadhelper#142] [Announcements]
_Version 8.2.0 broke stuff. Sorry about that 🙇. If you want to help, please join our [beta program](https://www.downloadhelper.net/w/Beta-versions)._

We are releasing version 8.2.1, which should address most of the issues. For now, it's Firefox only: https://www.downloadhelper.net/w/Beta-versions

Note: [please make sure your coapp is up to date](https://www.downloadhelper.net/w/CoApp-Installation).

Changelog:

- Proper naming in browser-download
- Ask for name in browser-download
- Respect Download Processor "Ask" value in pref
- fix clipboard issue on all browser
- fix MKV-only downloads
- fix EPerm error on Windows
- support M4V downloads
- copy audio manifest if no video manifest is present
- fix download duration going beyond 100%

**Comment by koolio888** [comment]
Still not completely download video beyond 100%. Tried all versions, beta or not from all 3 platforms. Cache memory cleared.
Problems for example: 
1. one of the video after downloaded 100%, only capture 1min 48sec out of the 1min 53 sec video.
2. another video, only capture 36 sec of the 43 sec.

All videos from the same website, missing around 7sec after downloaded.

**Reply by paulrouget** [reply]
Can you try this: https://github.com/aclap-dev/video-downloadhelper/discussions/236

**Reply by koolio888** [reply]
It does fix the truncated video problem. Thanks! Now that creates a new problem.
I recently bought the license for chrome version. It is showing License/Browser mismatch with your beta.
Will you please fix that for me?

**Reply by Mattmjs** [reply]
A license is browser specific, I believe your using the beta version for Firefox correct ?
So that means for chrome & also chrome beta version.

**Comment by paulrouget** [comment]
Yes! We are aware of that problem (happening on MPD content). It's high priority in our todolist.
